internal/playground/src/WalletPicker.tsx

```diff
@@ -37,7 +37,7 @@ export const availableChainsByWallet: Record<WalletOption, Chain[]> = {
   [WalletOption.OKX_MOBILE]: EVMChainList,
   [WalletOption.COINBASE_WEB]: EVMChainList,
   [WalletOption.KEPLR]: [Chain.Cosmos],
-  [WalletOption.KEYSTORE]: AllChainsSupported,
+  [WalletOption.KEYSTORE]: [...AllChainsSupported, Chain.Polkadot, Chain.Chainflip],
```

Comment:
Maybe we should export those sets of chains (EVMChains, CosmosChains, UTXOChains, SubstrateChains, AllChains) as consts from swapkit? 🤔